tests/test_unexpansion.py

```diff
@@ -324,10 +324,12 @@ def test_redundant_derivatives(self):
         assert len(get_arrays(op)) == 0
         assert op._profiler._sections['section0'].sops == 74
         exprs = FindNodes(Expression).visit(op)
-        assert len(exprs) == 6
+        assert len(exprs) == 5
```

Comment:
Note for reviewers:

this is a result of this tweak: https://github.com/devitocodes/devito/pull/2310/files#diff-a012e847c0c9841809763e4c55eb89ee0ee9b985614693494911e4ab32bda7f1R193-R194